### PR TITLE
Feature: Provide ability for the admin to filter by districts.

### PR DESCRIPTION
**Pull request by Roland-Sankara**
### What does this PR do? 

- Add the UI for the District Filter
- Create global state to manage to effect data updates by filter
    
### What does this PR solve?
Solves part of: https://outboxhub.atlassian.net/browse/GDU-13

### CODE REVIEW

File: src/components/SideNav.js [hunk 1]
```diff
@@ -100,15 +102,20 @@ export default class SideNav extends Component {
       district: district
     });
   }
+
+  static contextType = GlobalContext;
+
   render() {
+    console.log(this.context);
```
Comment (innyanzi) [inline src/components/SideNav.js:109]: Remove any console.logs

File: src/components/SideNav.js [hunk 2]
```diff
@@ -100,15 +102,20 @@ export default class SideNav extends Component {
...
     return (
       <div className={this.state.show ? "sideNav" : "sideNav hiddenMenu"}>
         <nav className="headerNav navbar navbar-default navbar-static-top">
           <div className="navbar-header">
             <a className="navbar-brand" href="/dashboard">
               <img alt="GETIN" src={require("../assets/images/Logo.png")} />
               <span className="brand">GetIN</span>
-              <span className="district">{this.state.district}</span>
+              {/* <span className="district">{this.state.district}</span> */}
```
Comment (innyanzi) [inline src/components/SideNav.js:117]: Remove any unused code.

**Review by innyanzi — CHANGES_REQUESTED** [review]
- Remove any console.logs and commented code

File: src/components/SideNav.js (hunk 2 above)
Comment (Roland-Sankara) [reply src/components/SideNav.js:117]: Alright, let me work on that

**Review by innyanzi — APPROVED** [review]

**Pull request merged**